src/query/insert.rs

```diff
@@ -148,18 +82,133 @@ where
         self.query.values_panic(values);
         self
     }
+}
+
+impl<A> Insert<A>
+where
+    A: ActiveModelTrait,
+{
+    /// On conflict
+    ///
+    /// on conflict do nothing
+    /// ```
+    /// use sea_orm::{entity::*, query::*, sea_query::OnConflict, tests_cfg::cake, DbBackend};
+    ///
+    /// let orange = cake::ActiveModel {
+    ///     id: ActiveValue::set(2),
+    ///     name: ActiveValue::set("Orange".to_owned()),
+    /// };
+    /// assert_eq!(
+    ///     cake::Entity::insert(orange)
+    ///         .on_conflict(
+    ///             OnConflict::column(cake::Column::Name)
+    ///                 .do_nothing()
+    ///                 .to_owned()
+    ///         )
+    ///         .build(DbBackend::Postgres)
+    ///         .to_string(),
+    ///     r#"INSERT INTO "cake" ("id", "name") VALUES (2, 'Orange') ON CONFLICT ("name") DO NOTHING"#,
+    /// );
+    /// ```
+    ///
+    /// on conflict do update
+    /// ```
+    /// use sea_orm::{entity::*, query::*, sea_query::OnConflict, tests_cfg::cake, DbBackend};
+    ///
+    /// let orange = cake::ActiveModel {
+    ///     id: ActiveValue::set(2),
+    ///     name: ActiveValue::set("Orange".to_owned()),
+    /// };
+    /// assert_eq!(
+    ///     cake::Entity::insert(orange)
+    ///         .on_conflict(
```

Comment:
OH, I get it now. It was shifted. And I suppose the exsiting impl wasn't actually changed. If that's the case, it'd be fine.